### ISSUE TITLE
~/.todorc configuration file

### ISSUE THREAD

**Issue by Zearin** [alecthomas/devtodo2#9]
From the README's “Not currently supported” list

**Comment by DenLilleMand** [comment]
@alecthomas I was thinking about making the configuration in JSON, now that the tasklist is in JSON i think it would make it more coherent, and if you could open this issue again, then i will refer to this in my PR when i get that far.

**Comment by DenLilleMand** [comment]
@alecthomas  and ... I was thinking about making something like a priority for options, so the options would be loaded and overriden in the following order:

1. Defaults
2. config from .todorc
3. cmd line options

And all of the options should be available in the cmd line and .todorc ...

What do you think about this??

**Comment by alecthomas** [comment]
Yes, seems reasonable to me! You can see an example of using kingpin with JSON config, in gometalinter.
